src/index.ts

```diff
@@ -33,6 +33,9 @@ interface ILernaProject {
   readonly useWorkspaces: boolean;
   readonly docsDirectory: string;
   readonly docgen?: boolean;
+  readonly taskCustomizations: TaskCustomizations;
```

Comment:
Made this public. Not sure whether this is right. May be best to make this private and expose it when there is a use case raised